lib/hashtable.cc

```diff
@@ -1502,4 +1488,31 @@ void Hashtable::extract_unique_paths(std::string seq,
         }
     }
 }
+
+
+void Hashtable::get_kmers(const std::string &s,
+                          std::vector<std::string> &kmers_vec) const
+{
+    if (s.length() < _ksize) {
+        return;
```

Comment:
In other cases where the sequence is shorter than K, we raise an exception; is this a case of letting an error pass silently?